### ISSUE TITLE
Maven: package target errors

### ISSUE THREAD

**Issue by javajawa** [edgemaster/DoCitten#6]
When running `mvn package` from either edgemaster/DoCitten@23bc4db using edgemaster/InternetRelayCats@d6577c3, I receive the following error:

```
[ERROR] Failed to execute goal on project internetrelaycats:
Could not resolve dependencies for project uk.co.harcourtprogramming:internetrelaycats:jar:1.0-SNAPSHOT:
Could not find artifact uk.co.harcourtprogramming:mewler:jar:tests:1.0-SNAPSHOT
```

(line breaks added for readability).

The issue is not present for `mvn compile` or `mvn test`.

Debug log from `mvn -e -X package`: [maven-output.txt](https://github.com/edgemaster/DoCitten/files/258905/maven-output.txt)

**Comment by IgnoredAmbience** [comment]
Yeah, I stopped once I got test and coveralls working. Packaging is the
next job.

**Comment by javajawa** [comment]
Oh, ok -- it looked to me like it _should_ have worked, but there again my maven experience is pretty much non-existent.

**Comment by IgnoredAmbience** [comment]
I'm doing some less-than-standard stuff with it, the general structure of
multi-package maven projects is not through submodule inclusion, but
multiple separate projects that are release-version controlled and packaged
into a repository for use by dependent projects. DoCitten doesn't fit that
too well at the moment, so I was working around it.
